financials/Funding request template.txt

```diff
@@ -0,0 +1,24 @@
+## Template for funding request
+This funding request template was made based on past experiences and debates on the Grin Forum about the necessity to have clear, 
+transparent and unambiguous information in funding requests. 
+This template is not intended to be exhaustive and only serves as a guideline to clearly communicate the minimal amount of information 
+needed for the GRIN community to assess requests for funding. Feel free to add and modify to this template to improve clarity and transparency. 
+However, let us aim to keep this funding template minimal and elegant like GRIN and the Mimble Wimble protocol itself. 
+For ease of comparison of funding requests, it is advised to provide these fields of information in the order given below.
+
+## Financial information
+�	Funding period: 
+o	(e.g. 3 months)
+�	Total amount of requested funding:
+o	State total amount of requested funding in this period as well as in the preceding funding period both in USD and local currency
+�	Requested funding per month and number FTE�s
+o	E.g. 1 FTE is equivalent to 40 hours per week
+o	Provide requested funding per month in USDs (based on 3 months average in Google) as well as in home currency
+o	Provide the % increase/decrease in hour rate compared to previous period based on the USD value. If increased, provide an explanation why the increase is requested, e.g. I do not ask for a raise since the value requested in my home currency is the same.
+�	Country from which you work:
+o	In case of residency in multiple countries, provide additional information (e.g. 0.5 FTE in Philippines, 0.5 FTE in US)
+o	If changed from previous period, provide explanation why the increase in hours is needed
+
+## Content information 

```

Comment:
This is perhaps the most important part of the request, i.e. "What is it that the person is proposing they would work on?". Maybe it could be expanded a bit further? 

Maybe we could propose an example of how to define this section? For example OKRs is a relatively simple and low-bureaucratic way to define goals and objectives. https://www.perdoo.com/the-ultimate-okr-guide/